scene/gui/slider.cpp

```diff
@@ -189,6 +189,35 @@ void Slider::_notification(int p_what) {
 			Size2i size = get_size();
 			double ratio = Math::is_nan(get_as_ratio()) ? 0 : get_as_ratio();
 
+			if (find_ticks) {
+				float step = get_step(), dist = get_max() - get_min();
+				int idist = dist;
+				if (step > dist) {
+					ticks = 0;
```

Comment:
I don't think it should directly modify the exposed property. There can be an internal structure or set of properties that are affected by both the fixed value and the computed value.

And it certainly shouldn't be done on a each draw call. This value is static as long as the number of steps doesn't change. So it should be computed once, when the relevant properties change, and only used during the drawing process.